### ISSUE TITLE
The loss will be nan when it is lower than 20

### ISSUE THREAD

**Issue by wrx33** [JY-112553/yolov1-keras-voc#4]
I have trained the model following the guidance. However, when the loss reduced below 20, it will be nan.
I have not figured out the reason.
